ledger/test-common/BUILD.bazel

```diff
@@ -109,6 +112,12 @@ da_scala_library(
         ),
         ("performance", []),
         ("package_management", []),
+        ("simple_package_party", []),
+        ("simple_package_optional", []),
+        ("simple_package_list", []),
+        ("simple_package_text_map", []),
+        ("simple_package_variant", []),
+        ("simple_package_tuple", []),
```

Comment:
As discussed earlier, these belong in their own Bazel file under _kvutils_; they're not intended to be used by anything else.

We also don't need to generate them per DAML-LF version, but only for the dev version.

Reply:
Sorry, answered below.